pkgs/development/python-modules/fastpair/default.nix

```diff
@@ -19,9 +19,6 @@ buildPythonPackage {
     scipy
   ];
 
-  # Does not support pytest 4 https://github.com/carsonfarmer/fastpair/issues/14
-  doCheck = false;
-
```

Comment:
Could we switch to use `pytestCheckHook`?

Reply:
ok.

Reply:
done